packages/validator/templates/OAS2OIS.json

```diff
@@ -0,0 +1,288 @@
+{
+	"__actions": [
+		{
+			"__insert": {
+				"__target": "oisFormat",
+				"__value": "1.0.0"
+			}
+		},
+		{
+			"__insert": {
+				"__target": "apiSpecifications.servers",
+				"__value": []
+			}
+		},
+		{
+			"__insert": {
+				"__target": "endpoints",
+				"__value": []
+			}
+		},
+		{
+			"__insert": {
+				"__target": "apiSpecifications.security",
+				"__value": {}
+			}
+		},
+		{
+			"__insert": {
+				"__target": "apiSpecifications.components.securitySchemes",
+				"__value": {}
+			}
+		},
+		{
+			"__insert": {
+				"__target": "apiSpecifications.paths",
+				"__value": {}
+			}
+		}
+	],
+	"info": {
+		"title": {
+			"__actions": [
+				{
+					"__copy": {
+						"__target": "title"
+					}
+				}
+			]
+		},
+		"version": {
+			"__actions": [
+				{
+					"__copy": {
+						"__target": "version"
+					}
+				}
+			]
+		},
+		"__ignore": {}
+	},
+	"servers": {
+		"__arrayItem": {
+			"url": {
+				"__regexp": "^(https?|ftp)://[^\\s/$.?#].[^\\s]*$",
```

Comment:
I don't see anything immediately wrong with the regex, but I checked the OAS specs and they say:
> Server URL format follows RFC 3986 and usually looks like this:
> ` scheme://host[:port][/path]`

And the RFC 3986 docs actually contain a regular expression:
https://tools.ietf.org/html/rfc3986#appendix-B

although we would ignore:
`
query     = $7
fragment  = $9
`

Maybe it would be safer to use the regex from the protocol document itself. What do you think, @bbenligiray, @Drgy?

Reply:
@bbenligiray are we to support the exact same server url formats as OAS?

E.g. OAS allows relative URLs as well as `localhost` (for example) and neither of these would be captured by Jakub's regex.

https://swagger.io/docs/specification/api-host-and-base-path/
^ see: "Server URL Format"

Reply:
I agree with using  https://tools.ietf.org/html/rfc3986#appendix-B omitting the part starting from query.
Let's only support absolute URLs
https://github.com/api3dao/api3-docs/commit/75ebe672addd1c6e4cc2e68bf47f06515161c0ed

Reply:
Since this should be updated in all templates I moved it into [appropriate issue](https://github.com/api3dao/airnode/issues/197)